cloudrun-malware-scanner/cloudbuild.yaml

```diff
@@ -0,0 +1,35 @@
+steps:
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'build'
+      - '-t'
+      - "${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:$COMMIT_SHA"
+      - '-t'
+      - '${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:latest'
+      - '-f'
+      - 'cloudrun-malware-scanner/Dockerfile'
+      - 'cloudrun-malware-scanner'
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'push'
+      - "${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:$COMMIT_SHA"
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'push'
+      - '${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:latest'
+  - name: '${_GCR_HOSTNAME}/google.com/cloudsdktool/cloud-sdk'
+    entrypoint: 'gcloud'
+    args:
+      - 'run'
+      - 'deploy'
+      - '${_SERVICE_NAME}'
+      - "--image=${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:$COMMIT_SHA"
+      - '--no-allow-unauthenticated'
```

Comment:
can remove this as well
we will configure this manually(or via terraform) 

again.. this should only update the artifact and not change anything else about this cloud run instance